identity-apps-core/apps/recovery-portal/src/main/webapp/password-reset-complete.jsp

```diff
@@ -74,6 +74,7 @@
     String newPassword = request.getParameter("reset-password");
     String callback = request.getParameter("callback");
     String spId = request.getParameter("spId");
+    String sp = request.getParameter("sp");
```

Comment:
et's encode these parameter values using OWASP Encoder; otherwise, these parameter usages can be exploited as XSS vulnerabilities.